src/theme/irisTheme.js

```diff
@@ -55,17 +57,20 @@ export const colors = {
   graphics: ['#41d3bd', '#a95adf'],
   border: {
     medium: '#c2c2c2',
+    primary: '#004658',
```

Comment:
As a consequence of the lack of organization in the theme, `colors.primary.border` already exists. Not that I disagree with moving the value here, but we do now have two different color values for `colors.primary.border` and `colors.border.primary`.

Trouble is, some of our colors are organized by color meaning ('primary'), some by usage ('border'). I guess this could all use some work.

Reply:
Changed to `colors.border.dark`.